recipes/jemalloc/all/conanfile.py

```diff
@@ -187,24 +194,24 @@ def _library_name(self):
         return libname
 
     def package(self):
-        self.copy(pattern="COPYING", src=self._source_subfolder, dst="licenses")
+        self.copy(pattern="COPYING", src=self.source_folder, dst="licenses")
         if self.settings.compiler == "Visual Studio":
             arch_subdir = {
                 "x86_64": "x64",
                 "x86": "x86",
             }[str(self.settings.arch)]
-            self.copy("*.lib", src=os.path.join(self._source_subfolder, "msvc", arch_subdir, self._msvc_build_type), dst=os.path.join(self.package_folder, "lib"))
-            self.copy("*.dll", src=os.path.join(self._source_subfolder, "msvc", arch_subdir, self._msvc_build_type), dst=os.path.join(self.package_folder, "bin"))
-            self.copy("jemalloc.h", src=os.path.join(self._source_subfolder, "include", "jemalloc"), dst=os.path.join(self.package_folder, "include", "jemalloc"), keep_path=True)
-            shutil.copytree(os.path.join(self._source_subfolder, "include", "msvc_compat"),
+            self.copy("*.lib", src=os.path.join(self.source_folder, "msvc", arch_subdir, self._msvc_build_type), dst=os.path.join(self.package_folder, "lib"))
+            self.copy("*.dll", src=os.path.join(self.source_folder, "msvc", arch_subdir, self._msvc_build_type), dst=os.path.join(self.package_folder, "bin"))
+            self.copy("jemalloc.h", src=os.path.join(self.source_folder, "include", "jemalloc"), dst=os.path.join(self.package_folder, "include", "jemalloc"), keep_path=True)
+            shutil.copytree(os.path.join(self.source_folder, "include", "msvc_compat"),
                             os.path.join(self.package_folder, "include", "msvc_compat"))
         else:
             autotools = self._configure_autotools()
             # Use install_lib_XXX and install_include to avoid mixing binaries and dll's
             autotools.make(target="install_lib_shared" if self.options.shared else "install_lib_static")
             autotools.make(target="install_include")
             if self.settings.os == "Windows" and self.settings.compiler == "gcc":
-                tools.rename(os.path.join(self.package_folder, "lib", "{}.lib".format(self._library_name)),
+                tools_legacy.rename(os.path.join(self.package_folder, "lib", "{}.lib".format(self._library_name)),
```

Comment:
I am surprised there's no files tool for this 🤔

Reply:
Actually, there is since Conan 1.37.0: https://docs.conan.io/en/latest/reference/conanfile/tools/files/basic.html#conan-tools-files-rename